server/models/index.ts

```diff
@@ -0,0 +1,93 @@
+// server/models.ts
+import { DataTypes, Model } from 'sequelize';
+import { sequelize } from './database';
+
+class User extends Model {}
```

Comment:
It looks like User/Conversation/Message are now defined twice? Should the definitions in this file be replaced with imports?

Reply:
@mentatbot please fix this